modules/claims_api/app/swagger/claims_api/v2/swagger.json

```diff
@@ -0,0 +1,599 @@
+                                "type": "string",
+                                "description": "The type of claim originally submitted",
+                                "enum": [
+                                  "Compensation",
+                                  "Compensation and Pension",
+                                  "Dependency",
+                                  "Administrative Review"
+                                ],
+                                "example": "Compensation"
+                              }
+                            }
+                          }
+                        }
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          },
+          "401": {
+            "description": "Unauthorized",
+            "content": {
+              "application/json": {
+                "example": {
+                  "errors": [
+                    {
+                      "title": "Not authorized",
+                      "detail": "Not authorized",
+                      "code": "401",
+                      "status": "401"
+                    }
+                  ]
+                },
+                "schema": {
+                  "required": [
+                    "errors"
+                  ],
+                  "properties": {
+                    "errors": {
+                      "type": "array",
+                      "items": {
+                        "additionalProperties": false,
+                        "required": [
+                          "title",
+                          "detail",
+                          "code",
+                          "status"
+                        ],
+                        "properties": {
+                          "title": {
+                            "type": "string",
+                            "description": "HTTP error title"
+                          },
+                          "detail": {
+                            "type": "string",
+                            "description": "HTTP error detail"
+                          },
+                          "code": {
+                            "type": "string",
+                            "description": "HTTP error code"
+                          },
+                          "status": {
+                            "type": "string",
+                            "description": "HTTP error code"
+                          }
+                        }
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          },
+          "404": {
+            "description": "Resource Not Found",
+            "content": {
+              "application/json": {
+                "example": {
+                  "errors": [
+                    {
+                      "title": "Resource not found",
+                      "detail": "Resource not found",
```

Comment:
nit: not critical here but i think when we get to the real implementation for this we should have something like "Veteran not found" for the detail here. i remember one of our interviews mentioned it was odd to have a 404 for an endpoint that is supposed to return a list and i agreed with him at the time. Just now realizing this situation should only occur if no Veteran is found with the passed in identifier.